spec.md

```diff
@@ -1047,6 +1047,13 @@ $foo->bar(
 );
 ```
 
+### 7.1 Function callable references
+
+A function may be referenced in a way that creates a closure out of it, by providing `...` in place of arguments.
```

Comment:
Need an example.

Reply:
There literally is one in the very next sentence. :smile:

Reply:
Seems I've missed a PHP feature :)